crates/nu-protocol/src/engine/jobs.rs

```diff
@@ -230,6 +245,7 @@ impl Job {
 pub struct FrozenJob {
     pub unfreeze: UnfreezeHandle,
     pub tag: Option<String>,
+    pub timestamp: DateTime<Utc>,
```

Comment:
I am not sure if using `chrono::DateTime` is the best choice here. I don't think it has the monotonicity guarantees of [`std::time::Instant`](https://doc.rust-lang.org/std/time/struct.Instant.html) which should preserve the order of jobs even if you are messing with timezone or clock settings while a job is frozen.

If we don't need to directly present it to users having the support for timezones etc. would be overkill.